cpp/ql/lib/semmle/code/cpp/XML.qll

```diff
@@ -32,12 +32,15 @@ class XMLLocatable extends @xmllocatable, TXmlLocatable {
   string toString() { none() } // overridden in subclasses
 }
 
+/** DEPRECATED: Alias for XmlLocatable */
+deprecated class XMLLocatable = XmlLocatable;
+
 /**
  * An `XMLParent` is either an `XMLElement` or an `XMLFile`,
  * both of which can contain other elements.
  */
-class XMLParent extends @xmlparent {
-  XMLParent() {
+class XmlParent extends @xmlparent {
+  XmlParent() {
     // explicitly restrict `this` to be either an `XMLElement` or an `XMLFile`;
```

Comment:
```suggestion
    // explicitly restrict `this` to be either an `XmlElement` or an `XmlFile`;
```

Reply:
The `XMLDTD` had also been missed by my patch (probably because of the all upper-case name), so I also updated that one.